<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptDialog</class>
 <widget class="QDialog" name="EncryptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>p2kontakt: Encrypt</string>
  </property>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Encrypting as &lt;user&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_encrypt">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_encrypted">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>251</width>
     <height>331</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_plaintext">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>281</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
